### ISSUE TITLE
Library/Boards Manager uninstall button not accessible via keyboard

### ISSUE THREAD

**Issue by per1234** [arduino/arduino-ide#1661]
### Describe the problem

The "**Library Manager**" and "**Boards Manager**" views provide the following important capabilities for management of sketch dependencies:

- Installation
- Update
- Uninstallation

The uninstall capability uses an odd interface where the "**INSTALLED**" label shown near the top right corner of the entries of dependencies that are currently installed on the user's system becomes an "**UNINSTALL**" button when the mouse pointer is hovered over the label.

An essential requirement for accessibility, as well as for the convenience of all users, is that all elements of the Arduino IDE UI be usable by keyboard alone.

🐛 It is impossible to uninstall libraries and boards platforms via the "**Library Manager**" and "**Boards Manager**" views using a keyboard.

### To reproduce

Attempt to uninstall any installed library or boards platform using the keyboard alone.

🐛 Even thought it is possible to select the "**INSTALLED**" label using the enhanced navigation capabilities of Windows [**Narrator**](https://support.microsoft.com/en-us/windows/complete-guide-to-narrator-e4397a0d-ef4f-b386-d8ae-c172f109bdb1) or [**NVDA**](https://www.nvaccess.org/) (in ["browse mode"](https://www.nvaccess.org/files/nvda/documentation/userGuide.html#BrowseMode)), it is impossible to trigger an uninstall using the keyboard.

### Expected behavior

The uninstall capability of Library Manager and Boards Manager is easy to access using a keyboard.

### Arduino IDE version

3a70547

### Operating system

Windows

### Operating system version

10

### Additional context

Related: https://github.com/arduino/arduino-ide/issues/360

### Issue checklist

- [X] I searched for previous reports in [the issue tracker](https://github.com/arduino/arduino-ide/issues?q=)
- [X] I verified the problem still occurs when using the latest [nightly build](https://github.com/arduino/arduino-ide#nightly-builds)
- [X] My report contains all necessary details
